Package:     squidguard
Subject:     Fix for clean target without syslog
Author:      Joachim Wiedorn <ad_debian@joonet.de>
Forwarded:   no
Last-Update: 2012-04-22

Simple fix for clean target without syslog for use with Debian.
---

diff -urN s09/src/main.cl s10/src/main.c
--- squidguard-1.5.orig/src/main.c	2011-10-01 20:49:00.000000000 +0200
+++ squidguard-1.5/src/main.c	2012-04-22 19:04:00.000000000 +0200
@@ -99,7 +99,7 @@ int main(argc, argv, envp)
   progname = argv[0];
   globalPid = getpid();
 #ifdef USE_SYSLOG
-openlog("squidGuard", LOG_PID | LOG_NDELAY | LOG_CONS, LOG_LOCAL2);
+openlog("squidGuard", LOG_PID | LOG_NDELAY | LOG_CONS, LOG_);
 #endif
   while ((ch = getopt(argc, argv, "hbduPC:t:c:v")) != EOF)
     switch (ch) {
